decidim-budgets/app/controllers/decidim/budgets/projects_controller.rb

```diff
@@ -54,7 +54,7 @@ def default_filter_params
       end
 
       def default_filter_status_params
-        voting_finished? ? %w(selected) : %w(all)
+        voting_finished? && budget.projects.selected.any? ? %w(selected) : %w(all)
       end
```

Comment:
I think that if we move the condition to its own method we can improve the legibility just a bit
```suggestion
        show_selected_budgets? ? %w(selected) : %w(all)
      end
      
      def show_selected_budgets?
        voting_finished? && budget.projects.selected.any
      end
```

Reply:
Thanks. Done.